mandatory/1-fix-functions.js

```diff
@@ -10,30 +10,28 @@
 
 */
 
-function mood() {
+function mood(answer) {
   let isHappy = true;
 
-  if (isHappy) {
+  if (answer === isHappy) {
```

Comment:
An alternative solution to this, would be to also make the function argument "function mood(isHappy)"